flink-runtime/src/main/java/org/apache/flink/runtime/executiongraph/DefaultExecutionGraph.java

```diff
@@ -1042,8 +1041,7 @@ private boolean transitionState(JobStatus current, JobStatus newState, Throwable
                     newState,
                     error);
 
-            stateTimestamps[newState.ordinal()] = System.currentTimeMillis();
```

Comment:
Was this line removed on purpose? Seems unrelated to removing the error

Reply:
No that was an accident when transferring the changes from WIP branch.